### ISSUE TITLE
ios build complains about use of Ad ID

### ISSUE THREAD

**Issue by laurentS** [mapswipe/mapswipe#527]
**Describe the bug**
When building the app in XCode, I see the following warning:
```
RNFBAnalytics: Using default Firebase/Analytics with Ad Ids. May require App Tracking Transparency. Not allowed for Kids apps.
RNFBAnalytics: You may set variable `$RNFirebaseAnalyticsWithoutAdIdSupport=true` in Podfile to use analytics without ad ids.
```

**App version**
ios build only, first noticed the problem after upgrading to RN 0.64.2.

**Comment by laurentS** [comment]
Probably related to #485

**Comment by laurentS** [comment]
closed with #544
